libavfilter/f_ebur128.c

```diff
@@ -167,6 +168,7 @@ static const AVOption ebur128_options[] = {
         { "info",    "information logging level", 0, AV_OPT_TYPE_CONST, {.i64 = AV_LOG_INFO},    INT_MIN, INT_MAX, A|V|F, "level" },
         { "verbose", "verbose logging level",     0, AV_OPT_TYPE_CONST, {.i64 = AV_LOG_VERBOSE}, INT_MIN, INT_MAX, A|V|F, "level" },
     { "metadata", "inject metadata in the filtergraph", OFFSET(metadata), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, A|V|F },
+    { "gate", "use the gated loudness measurement", OFFSET(gate_measurement), AV_OPT_TYPE_BOOL, {.i64 = 1}, 0, 1, A|F },
```

Comment:
Should the description be "use gating for integrated loudness measurement" or "use ITU-BS.1770-2+ gating for integrated loudness measurement"?